### PR TITLE
feat: Reporting

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: site/utils/exportUtils.ts [hunk 1]
```diff
@@ -73,5 +73,25 @@ export const getExportSchema = (): Schema<ExportDisruptionData> => {
...
         },
+        {
+            column: "Planned/Unplanned",
+            type: String,
+            value: (objectData: ExportDisruptionData) => objectData.disruptionType,
+        },
+        {
+            column: "Date added",
+            type: String,
+            value: (objectData: ExportDisruptionData) => objectData.creationTime,
```
Comment (DeannaSharma) [inline site/utils/exportUtils.ts:84]: Old disruption don't have a creation time so when I export the data a lot of disruptions have N/A under "date added"

In the SIRI generator we have a function called `getDisruptionCreationTime` where if there is not a creation time we use the disruption history to get when the disruption was created. Can we reuse this function to populate the date added column?

Comment (alainajaffer) [reply site/utils/exportUtils.ts:84]: done

File: site/pages/api/get-all-disruptions/__snapshots__/get-all-disruptions.test.ts.snap [hunk 2]
```diff
@@ -9,6 +9,7 @@ exports[`getAllDisruptions > formatSortedDisruptions > correctly formats disrupt
   "displayId": "8fg3ha",
   "disruptionReason": "accident",
   "disruptionType": "planned",
+  "history": undefined,
```
Comment (DeannaSharma) [inline site/pages/api/get-all-disruptions/__snapshots__/get-all-disruptions.test.ts.snap:12]: do we expect this to be undefined? we might need to update the mock data the test is using to include a mock history item.

Comment (alainajaffer) [reply site/pages/api/get-all-disruptions/__snapshots__/get-all-disruptions.test.ts.snap]: done

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**